plone/app/relationfield/tests.zcml

```diff
@@ -0,0 +1,16 @@
+<configure
```

Comment:
Hi @tisto 

OK, I'll adjust the tests. But I have still a short question: Is there a way to register an behavior without any zcml or include the zcml  in the testsetup itself. Because the bug I'll fix, only happens on behavior with marker interface including a `RelationList` field.